images/nixos-image.nix

```diff
@@ -21,36 +25,82 @@ nixpkgs.lib.nixosSystem {
           # of an iso, but works nonetheless.
           "${modulesPath}/installer/cd-dvd/installation-cd-minimal-new-kernel-no-zfs.nix"
         ];
-        system.stateVersion = "25.05";
-
-        nix.enable = false;
 
-        boot.kernelParams = [
-          "console=ttyS0"
-          "earlyprintk=ttyS0"
+        boot.initrd.availableKernelModules = [
+          "virtio_blk"
+          "virtio_pci"
         ];
-
+        boot.initrd.kernelModules = [ "virtio_net" ];
+        boot.initrd.systemd.enable = false;
```

Comment:
Just out of interest: What is the consequence of setting this to false?

Reply:
less flexibility for stuff in the initrd and the initrd size (thus, the iso size) is smaller

<img width="1447" height="526" alt="image" src="https://github.com/user-attachments/assets/8a4ddd5d-6b04-4882-b551-eb7a9e52357a" />